index.js

```diff
@@ -262,18 +266,21 @@ function markDirty(tome, dirtyAt, was) {
 
 function diff(tome, op, val, chain, pair, was) {
 	var root = tome.__root__;
-	tome.__root__.__version__ += 1;
 
-	if (!chain) {
-		chain = Tome.buildChain(tome);
-	}
+	if (root.__diffEnabled__) {
+		root.__version__ += 1;
 
-	var newOp = { chain: chain, op: op };
-	if (val !== undefined) {
-		newOp.val = val;
-	}
+		if (!chain) {
+			chain = Tome.buildChain(tome);
+		}
 
-	root.__diff__.push(newOp);
+		var newOp = { chain: chain, op: op };
+		if (val !== undefined) {
+			newOp.val = val;
+		}
+
+		root.__diff__.push(newOp);
+	}
```

Comment:
The only thing that changed here was that a block of code was moved into an if-statement. Side-by-side diff will make this clear.